### PR TITLE
Added documentation for matplotlib.pyplot.scatter()

### LINKED ISSUE #1662: [Term Entry] Matplotlib `pyplot`: `scatter`
### Reason/inspiration (optional)

Please write a new entry under:  content/matplotlib/concepts/pyplot/terms/ about the [`.scatter()`](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.scatter.html) method in Python.

A "Syntax" section with pseudo code snippet.
An "Example" section with a Python code snippet. If images are included, they should go in the media/ folder.

To get started with the PR and entry file:

Under content/matplotlib/concepts/pyplot/terms/, create a new folder called `scatter`. Then create a new file called `scatter.md` under content/matplotlib/concepts/pyplot/terms/`scatter`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by alekseymor**
### Description

Added documentation for the scatter() function of matplotlib library.
Closes #1662 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+Title: '.scatter()
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:2]:
```suggestion
Title: '.scatter()'
```

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Description: 'Creates a scatter plot of x vs. y values.'
+Subjects:
+ - 'Data Science'
+ - 'Data Visualization'
+Tags:
+ - 'Graphs'
+ - ' Libraries'
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:9]:
```suggestion
 - 'Libraries'
```

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+```py
+import matplotlib.pyplot as plt
+
+x1 = [5, 13, 21, 28, 31, 34, 39, 44, 49]
+y1 = [14, 28, 44, 56, 67, 53, 47, 30, 11]
+
+plt.scatter(x1, y1)
+plt.show()
+```
+Output:
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:53]:
```suggestion

Output:

```
Added line breaks

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import matplotlib.pyplot as plt
+
+x2 = [11, 22, 33, 44, 55]
+y2 = [11, 22, 33, 44, 55]
+
+plt.scatter(x2, y2, s=150, c='#88c988', linewidth=3, marker='p' , edgecolor='#175E17', alpha=0.75)
+plt.show()
+```
+
+Output:
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:66]:
```suggestion
Output:

```
Added line break.

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+- `marker`: The marker style.
+- `cmap`: A Colormap instance used to map scalar data to colors.
+- `norm`: Normalization method used to scale scalar data to the [0, 1] range before mapping. Linear scaling is default.
+- `vmin` and `vmax`: Define the data range that the colormap covers (if norm is not specified).
+- `alpha`: Marker blending value, between 0 (transparent) and 1 (opaque).
+- `linewidths`: The linewidth of the marker edge.
+- `edgecolors`: The edge color of the marker.
+- `plotnonfinite`: Boolean value determining whether to plot nonfinite (`inf`, `-inf`, `nan`) values. Default is `False`.
+
+## Example
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:40]:
```suggestion
## Examples
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@alekseymor I did a first review and found this a pretty solid entry. My comments are mostly typos and a few formatting issues.

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+Both 'x' and 'y' parameters are required, and represent float or array-like objects. Other parameters are optional and vary plot features like marker size and/or color.
+
+`.scatter()` takes the following arguments:
+
+- `x` and `y`: Positional arguments of type float or array.
+- `s`: A float or an array (of size equal to x or y) specifying marker size.
+- `c`: An array or list specifying marker color.
+- `marker`: The marker style.
+- `cmap`: A Colormap instance used to map scalar data to colors.
+- `norm`: Normalization method used to scale scalar data to the [0, 1] range before mapping. Linear scaling is default.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:33]:
```suggestion
- `norm`: Normalization method used to scale scalar data to a range (0 to 1) before mapping. Linear scaling is default.
```

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+- `vmin` and `vmax`: Define the data range that the colormap covers (if norm is not specified).
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:34]:
```suggestion
- `vmin` and `vmax`: Sets the data range for the colormap (if norm is not specified).
```

Comment (caupolicandiaz) [reply content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:34]: Please be careful not repurpose descriptions in other documentation. All entry content should be original.

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+- `alpha`: Marker blending value, between 0 (transparent) and 1 (opaque).
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:35]:
```suggestion
- `alpha`: Sets the transparency value of markers (0: transparent, 1: opaque).
```

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+- `linewidths`: The linewidth of the marker edge.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:36]:
```suggestion
- `linewidths`: Sets the linewidth of the marker edge.
```

File: content/matplotlib/concepts/pyplot/terms/scatter/scatter.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+- `edgecolors`: The edge color of the marker.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/scatter/scatter.md:37]:
```suggestion
- `edgecolors`: Sets the edge color of the marker.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@alekseymor I've added my comments. Please update and tag me when you've finished making changes. Thanks

**Comment by alekseymor** [comment]
Thanks for the feedback @caupolicandiaz. I've made the changes you requested. Back to you :)

**Review by caupolicandiaz — APPROVED** [review]
@alekseymor Looks good.

**Comment by SSwiniarski** [comment]
@alekseymor Congrats on your new contribution. The live link is here:https://www.codecademy.com/resources/docs/matplotlib/pyplot/scatter

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
image

2.) Then log out and log back into Codecademy using your GitHub login:
image

**Pull request merged**